package.json

```diff
@@ -22,23 +22,24 @@
   },
   "homepage": "https://github.com/electron-userland/electron-download#readme",
   "dependencies": {
-    "debug": "^2.2.0",
+    "debug": "^3.0.0",
     "env-paths": "^1.0.0",
-    "fs-extra": "^2.0.0",
+    "fs-extra": "^4.0.1",
     "minimist": "^1.2.0",
-    "nugget": "^2.0.0",
+    "nugget": "^2.0.1",
     "path-exists": "^3.0.0",
-    "rc": "^1.1.2",
-    "semver": "^5.3.0",
-    "sumchecker": "^2.0.1"
+    "rc": "^1.2.1",
+    "semver": "^5.4.1",
+    "sumchecker": "^2.0.2"
```

Comment:
> unless it's necessary for a new feature/bugfix.

Because `^` is used here, it means that users will get any new versions. I like ideas described in the https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/ 

I believe that it is better to specify new version and more recent if possible. It doesn't harm at all. It is only one changed symbols (Of course, no one insist to publish new module version only because some dependency has a new minor version.) When module will be installed — package tool in any case will  resolve to latest allowed version.

As you can see, first commit in this PR was failed due to NPM bug (cache). That's why yarn is used on CI (reliable and fast) and I prefer to always just use `npm-check-updates -a` (and read changelings, of course).